Pokemon Showdown Enhanced Tooltips/js/showPokemonTooltip.js

```diff
@@ -612,7 +638,10 @@ ShowdownEnhancedTooltip.showPokemonTooltip = function showPokemonTooltip(clientP
         for (var _i6 = 0, _clientPokemon$moveTr = clientPokemon.moveTrack; _i6 < _clientPokemon$moveTr.length; _i6++) {
 						var _row = _clientPokemon$moveTr[_i6];
 						var move = Dex.getMove(_row[0]);
-            text += this.getPPUseText(_row) + ' Base power: ' + move.basePower + ' ' + Dex.getTypeIcon(move.type) + '<br />';
+            text += this.getPPUseText(_row) + ' Base power: ' + move.basePower + ' ' +
+							Dex.getTypeIcon(move.type) + ' ' +
+							`<img src="${Dex.resourcePrefix}sprites/categories/${move.category}.png" alt="${move.category}" />` +
```

Comment:
This is the same code they use in the Showdown Client to get category sprites. There doesn't appear to be a nice method like the `getTypeIcon` above.